influxdbwriter does not compile on Windows

https://build.icinga.com/job/icinga2-windows-package/arch=x86_64,label=windows/226/console

```
C:\jenkins\workspace\icinga2-windows-package\arch\x86_64\label\windows\lib\perfdata/influxdbwriter.cpp(231): error C2440: '<function-style-cast>' : cannot convert from 'bool' to 'icinga::String' [C:\jenkins\workspace\icinga2-windows-package\arch\x86_64\label\windows\build\lib\perfdata\perfdata.vcxproj]
          No constructor could take the source type, or constructor overload resolution was ambiguous
```

--- a/lib/perfdata/influxdbwriter.cpp
+++ b/lib/perfdata/influxdbwriter.cpp
@@ -228,7 +228,7 @@ String InfluxdbWriter::FormatInteger(int val)
 
 String InfluxdbWriter::FormatBoolean(bool val)
 {
-	return String(val);
+	return val ? "true" : "false";
 }
 
 void InfluxdbWriter::SendPerfdata(const Dictionary::Ptr& tmpl, const Checkable::Ptr& checkable, const CheckResult::Ptr& cr, double ts)

